### DISCUSSION TITLE
Nerd MINER / Info for more Crypto ?

### DISCUSSION THREAD

**Discussion by cmueller79** [BitMaker-hub/NerdMiner_v2#462] [Ideas]
It would be great if the Nerdminer were also available for other cryptos, not necessarily because it brings a lot,  but simply because I think the DISPLAY information is very nice and would like to build a few more Nerdminers on my monitor.  :) 
get it for KASPA ?

Thanks, great project

**Comment by MyOwn2C** [comment]
Source code is available for anyone to adapt to their own use if you are capable
